scripts/package.js

```diff
@@ -65,7 +66,8 @@ Promise.all(resources.map(deflate)).then(() => {
         '--without-inspector',
         '--without-etw',
         '--with-snapshot',
-      ].concat(isWindows ? ['--openssl-no-asm'] : []),
+      ].concat(isWindows ? ['--openssl-no-asm'] : [])
+      .concat(staticBinary ? ['--fully-static'] : []),
```

Comment:
I think this can be moved to the `else` branch above since we're never gonna build a static binary on windows.

Reply:
aren't this `concat`? I thought we were adding stuff ... not sure I understand what you mean...

Reply:
was actually happy that it looked quite elegant

Reply:
```js
.concat(isWindows ? ['--openssl-no-asm'] : (staticBinary ? ['--fully-static'] : []))
```

Reply:
gotcha